behavior.c

```diff
@@ -397,7 +398,248 @@ static void pairing_tick(void) {
 		dbnc = 0;
 	}	
 	
-}	
+}
+
+static void setting_tick(void) {
+#define SET_MENU (-1)
```

Comment:
for to be there I don't know...
indention was made automatically by a "format" tools of mplab X